nethttp/client.go

```diff
@@ -197,54 +198,79 @@ func (h *Tracer) clientTrace() *httptrace.ClientTrace {
 
 func (h *Tracer) getConn(hostPort string) {
 	ext.HTTPUrl.Set(h.sp, hostPort)
-	h.sp.LogEvent("Get conn")
+	h.sp.LogFields(log.String("event", "GetConn"))
 }
 
 func (h *Tracer) gotConn(info httptrace.GotConnInfo) {
 	h.sp.SetTag("net/http.reused", info.Reused)
 	h.sp.SetTag("net/http.was_idle", info.WasIdle)
-	h.sp.LogEvent("Got conn")
+	h.sp.LogFields(log.String("event", "GotConn"))
 }
 
 func (h *Tracer) putIdleConn(error) {
-	h.sp.LogEvent("Put idle conn")
+	h.sp.LogFields(log.String("event", "PutIdleConn"))
 }
 
 func (h *Tracer) gotFirstResponseByte() {
-	h.sp.LogEvent("Got first response byte")
+	h.sp.LogFields(log.String("event", "GotFirstResponseByte"))
 }
 
 func (h *Tracer) got100Continue() {
-	h.sp.LogEvent("Got 100 continue")
+	h.sp.LogFields(log.String("event", "Got100Continue"))
 }
 
 func (h *Tracer) dnsStart(info httptrace.DNSStartInfo) {
-	h.sp.LogEventWithPayload("DNS start", info.Host)
+	h.sp.LogFields(
+		log.String("event", "DNSStart"),
+		log.String("host", info.Host),
+	)
 }
 
 func (h *Tracer) dnsDone(httptrace.DNSDoneInfo) {
-	h.sp.LogEvent("DNS done")
+	h.sp.LogFields(log.String("event", "DNSDone"))
 }
 
 func (h *Tracer) connectStart(network, addr string) {
-	h.sp.LogEventWithPayload("Connect start", network+":"+addr)
+	h.sp.LogFields(
+		log.String("event", "ConnectStart"),
+		log.String("network", network),
+		log.String("addr", addr),
+	)
 }
 
 func (h *Tracer) connectDone(network, addr string, err error) {
-	h.sp.LogEventWithPayload("Connect done", network+":"+addr)
+	if err != nil {
+		h.sp.LogFields(
+			log.String("event", "ConnectDone"),
+			log.String("network", network),
+			log.String("addr", addr),
+			log.Error(err),
+		)
+	} else {
+		h.sp.LogFields(
+			log.String("event", "ConnectDone"),
+			log.String("network", network),
+			log.String("addr", addr),
+		)
+	}
 }
 
 func (h *Tracer) wroteHeaders() {
-	h.sp.LogEvent("Wrote headers")
+	h.sp.LogFields(log.String("event", "WroteHeaders"))
 }
 
 func (h *Tracer) wait100Continue() {
-	h.sp.LogEvent("Wait 100 continue")
+	h.sp.LogFields(log.String("event", "Wait100Continue"))
 }
 
 func (h *Tracer) wroteRequest(info httptrace.WroteRequestInfo) {
 	if info.Err != nil {
+		h.sp.LogFields(
+			log.String("event", "WroteRequest"),
```

Comment:
I think event should be `error` here - see https://github.com/opentracing/specification/blob/master/semantic_conventions.md#log-fields-table